examples/data/v5/testrecords/testrecord-v1.0.0.json

```diff
@@ -0,0 +1,27 @@
+{
+  "mappings": {
+    "testrecord-v1.0.0": {
+      "properties": {
+        "control_number": {
+          "type": "string"
```

Comment:
This is ok with ES5, but fires a warning. I'm not sure if we should explicitly change ES5 mappings to use the correct field type for `string` (`text`, unless there was a `index: not_analyzed`, in which case it should be `keyword`)